resources/integration_test/tests/query.yaml

```diff
@@ -0,0 +1,293 @@
+    - [ John, 84.5, 3 ]
+    - [ Jane, 84.5, 3 ]
+    - [ Mike, 84.5, 3 ]
+    - [ Mary, 84.5, 3 ]
+    - [ Pete, 84.5, 3 ]
+    - [ John, 97.4, 3 ]
+    - [ Jane, 97.4, 3 ]
+    - [ Mike, 97.4, 3 ]
+    - [ Mary, 97.4, 3 ]
+    - [ Pete, 97.4, 3 ]
+    - [ John, 89.3, 3 ]
+    - [ Jane, 89.3, 3 ]
+    - [ Mike, 89.3, 3 ]
+    - [ Mary, 89.3, 3 ]
+    - [ Pete, 89.3, 3 ]
+    - [ John, 91.9, 3 ]
+    - [ Jane, 91.9, 3 ]
+    - [ Mike, 91.9, 3 ]
+    - [ Mary, 91.9, 3 ]
+    - [ Pete, 91.9, 3 ]
+    - [ John, 87.5, 5 ]
+    - [ Jane, 87.5, 5 ]
+    - [ Mike, 87.5, 5 ]
+    - [ Mary, 87.5, 5 ]
+    - [ Pete, 87.5, 5 ]
+    - [ John, 74.4, 5 ]
+    - [ Jane, 74.4, 5 ]
+    - [ Mike, 74.4, 5 ]
+    - [ Mary, 74.4, 5 ]
+    - [ Pete, 74.4, 5 ]
+    - [ John, 80.1, 5 ]
+    - [ Jane, 80.1, 5 ]
+    - [ Mike, 80.1, 5 ]
+    - [ Mary, 80.1, 5 ]
+    - [ Pete, 80.1, 5 ]
+    - [ John, 83.3, 5 ]
+    - [ Jane, 83.3, 5 ]
+    - [ Mike, 83.3, 5 ]
+    - [ Mary, 83.3, 5 ]
+    - [ Pete, 83.3, 5 ]
+    - [ John, 78.5, 5 ]
+    - [ Jane, 78.5, 5 ]
+    - [ Mike, 78.5, 5 ]
+    - [ Mary, 78.5, 5 ]
+    - [ Pete, 78.5, 5 ]
+    row_count: 45
+    ordered: false
+    expected_sql_type: [ LONGVARCHAR, DOUBLE, INTEGER ]
+    expected_catalog_name: [ '', '', '' ]
+    expected_column_class_name: [ java.lang.String, double, int ]
+    expected_column_label: [ name, score, testid ]
+    expected_column_display_size: [ 0, 15, 10 ]
+    expected_precision: [ 0, 15, 10 ]
+    expected_scale: [ 0, 15, 0 ]
+    expected_schema_name: [ '', '', '' ]
+    expected_is_auto_increment: [ false, false, false ]
+    expected_is_case_sensitive: [ true, false, false ]
+    expected_is_currency: [ false, false, false ]
+    expected_is_definitely_writable: [ false, false, false ]
+    expected_is_nullable: [ columnNullable, columnNullable, columnNullable ]
+    expected_is_read_only: [ true, true, true ]
+    expected_is_searchable: [ true, true, true ]
+    expected_is_signed: [ false, true, true ]
+    expected_is_writable: [ false, false, false ]
+
+  - description: subquery
+    db: integration_test
+    sql: SELECT studentid FROM (SELECT * FROM class WHERE enrolled = TRUE) AS foo
+    ordered: false
+    skip_reason: SQL-659
+
+  - description: count_from_grades
+    db: integration_test
+    sql: SELECT COUNT(studentid) FROM grades
+    ordered: false
+    skip_reason: SQL-658
+
+  - description: count_star_from_grades
+    db: integration_test
+    sql: SELECT COUNT(*) As C FROM grades
+    ordered: false
+    skip_reason: SQL-658
+
+  - description: count_from_class
+    db: integration_test
+    sql: SELECT COUNT(studentid) FROM class
+    ordered: false
+    skip_reason: SQL-658
+
+  - description: average
+    db: integration_test
+    sql: SELECT avg(score) FROM grades where testid=5
+    ordered: false
+    skip_reason: SQL-659
+
+  - description: max
+    db: integration_test
+    sql: SELECT max(score) FROM grades
+    ordered: false
+    skip_reason: SQL-659
```

Comment:
What JDBC-specific feature are we testing by having separate cases for these aggregation functions?